autospotting.go

```diff
@@ -30,8 +30,15 @@ func main() {
 func run() {
 	log.Println("Starting autospotting agent, build:", Version)
 
-	log.Printf("Parsed command line flags: regions='%s' min_on_demand_number=%d min_on_demand_percentage=%.1f",
-		conf.Regions, conf.MinOnDemandNumber, conf.MinOnDemandPercentage)
+	log.Printf("Parsed command line flags: "+
+		"regions='%s' "+
+		"min_on_demand_number=%d "+
+		"min_on_demand_percentage=%.1f "+
+		"keep_instance_type=%t",
```

Comment:
This is a different feature.  We want to keep the instance type 'exactly' of what it is. Do we need to implement in this PR? I believe we should open another PR, feature for that.